src/data_management/wms_layers/WmsLayerForm.tsx

```diff
@@ -0,0 +1,357 @@
+              value: dataset.slug,
+            }
+          })}
+          valueChanged={(value: any) => {
+            console.log("datasets 123", value);
+            handleValueChange('datasets', value)
+          }}
+          validated
+        />
+        <span className={formStyles.FormFieldTitle}>
+          2: Data
+        </span>
+        <TextInput
+          title={'WMS Url *'}
+          name={'wms_url'}
+          placeholder={'http://example.com'}
+          value={values.wms_url as string}
+          valueChanged={handleInputChange}
+          clearInput={clearInput}
+          validated={!minLength(3, values.wms_url as string)}
+          errorMessage={minLength(3, values.wms_url as string)}
+          triedToSubmit={triedToSubmit}
+        />
+        <TextInput
+          title={'Slug *'}
+          name={'slug'}
+          placeholder={'Slug for this WMS'}
+          value={values.slug as string}
+          valueChanged={handleInputChange}
+          clearInput={clearInput}
+          validated={!minLength(3, values.slug as string)}
+          errorMessage={minLength(3, values.slug as string)}
+          triedToSubmit={triedToSubmit}
+        />
+        <TextInput
+          title={'Download Url'}
+          name={'download_url'}
+          placeholder={'http://example.com'}
+          value={values.download_url as string}
+          valueChanged={handleInputChange}
+          clearInput={clearInput}
+          validated={true}
+          // errorMessage={minLength(3, values.name as string)}
+          triedToSubmit={triedToSubmit}
+        />
+        <TextInput
+          title={'Legend Url'}
+          name={'legend_url'}
+          placeholder={'http://example.com'}
+          value={values.legend_url as string}
+          valueChanged={handleInputChange}
+          clearInput={clearInput}
+          validated={true}
+          triedToSubmit={triedToSubmit}
+        />
+        <TextInput
+          title={'Get Feature Url'}
+          name={'get_feature_info_url'}
+          placeholder={'http://example.com'}
+          value={values.get_feature_info_url as string}
+          valueChanged={handleInputChange}
+          clearInput={clearInput}
+          validated={true}
+          triedToSubmit={triedToSubmit}
+        />
+        <CheckBox
+          title={'Tiled'}
+          name={'tiled'}
+          value={values.tiled as boolean}
+          valueChanged={bool => handleValueChange('tiled', bool)}
+        />
+        <MinMaxZoomField
+          name={'minMaxZoom'}
+          value={{
+            minZoom: values.min_zoom as number,
+            maxZoom: values.max_zoom as number,
+          }}
+          valueChanged={(value:MinMax) => { 
+            if (values.min_zoom !== value.minZoom) {
+              handleValueChange('min_zoom', value.minZoom);
+            } else {
+              handleValueChange('max_zoom', value.maxZoom);
+            }
+          }}
+          triedToSubmit={triedToSubmit}
+        />
```

Comment:
I am not convinced with this approach for the min max field. I believe we better make 2 separate fields for them.